# curlopt-constants.c:128:49: error: 'CURL_STRICTER' undeclared (first use in this function)
             if (strEQ(name, "STRICTER")) return CURL_STRICTER;

# Since Curl >= 7.50.2: curl.h: make CURL_NO_OLDIES define CURL_STRICTER
# From https://github.com/curl/curl/commit/5fce88aa8c1256486377471186fc353c50d610b1

diff -Nru a/Curl.xs b/Curl.xs
--- a/Curl.xs	2014-02-21 17:08:30.000000000 +0100
+++ b/Curl.xs	2016-09-11 09:16:23.744957442 +0200
@@ -10,6 +10,7 @@
  * Software is furnished to do so, under the terms of the MIT license.
  */
 #define PERL_NO_GET_CONTEXT
+#define CURL_STRICTER
 #include "EXTERN.h"
 #include "perl.h"
 #include "XSUB.h"
